### ISSUE TITLE
King County COVID19 - Bug: Menus are not keyboard accessible

### ISSUE THREAD

**Issue by lamarjordanux** [a11ySea/testing#3]
## Issue Summary

Keyboard users who don't rely on a screen reader cannot open the menus in the main navigation (How do I, Services, etc.) 
Also the menus are announced as links instead of menus. 

## Steps to reproduce

1. Go to home page with a screen reader on
2. Tab to the How Do I menu
3. Activate the menu with Enter key

### Expected behavior

The menu open when you hit the Enter key and the arrow keys navigate into the menu items.

### Actual behavior

- The link is activated, and the menu doesn't open.

### Device Information:

 - OS: Mac
 - Browser : Chrome

### Screen Reader or AT Information

 - VoiceOver

### Current Code
```
(code snippet)
```

### Suggested Code
```
(code snippet)
```

# Specifications

- **WCAG Principal:**
- **WCAG SC:**
- **Severity:**

## Additional References

- Link to any articles or specs that give further guidance

**Comment by alboss** [comment]
This is probably endemic to the entire content management system. It isn't something the webmaster for Public Health can address. I'll see if we can get the right people on it.

Al
